vochain/scrutinizer/scrutinizer.go

```diff
@@ -106,6 +106,7 @@ type Scrutinizer struct {
 // VoteWithIndex holds a Vote and a txIndex. Model for the VotePool.
 type VoteWithIndex struct {
 	vote    *models.Vote
+	voterID []byte
```

Comment:
Do we actually need to store voterID here?

Reply:
Yes because this model is used on the VoteIndexPool (holds the list of votes that will be persisted on the database).